webant/webant.py

```diff
@@ -18,7 +18,52 @@
 from webserver_utils import gevent_run
 
 
-def create_app(configfile=None):
+class LibreantCoreApp(Flask):
+    def __init__(self, import_name):
+        super(LibreantCoreApp, self).__init__(import_name)
+        self.config.update({
+            'BOOTSTRAP_SERVE_LOCAL': True,
+            'DEBUG': True,
+            'PRESET_PATHS': [],  # defaultPreset should be loaded as default?
+            'FSDB_PATH': "",
+            'AGHERANT_DESCRIPTIONS': [],
```

Comment:
> ok, corrected. I'm not sure that this is what you mean, but I am sure that this is what I mean :)

This is not what I mean.
The problem is that you have reorganized the code to make it more modularized but you did not the same with configuration stuff.

The big question is:
Why ```LibreantCoreApp``` handle default setting values for something that it will never use?
For example ```AGHERANT_DESCRIPTIONS``` will be used only by the sub-class ```LibreantViewApp``` but the default value is initialized in ```LibreantCoreApp```.

The pattern that I suggest is something like this:

```LibreantCoreApp``` make a first call to ```config.update()``` to initialize default values that are going to be used in this class (```DEBUG```,```FSDB_PATH```,```SECRET_key``` etc...).
Then it make another call to ```config.update()``` using as paramter the one received in the constructor in order to set default values for stuff that are going to be used in the child-classes.

Using this pattern ```LibreantCoreApp``` will start like this:
```python
def __init__(self, import_name, defConf={}):
   super(LibreantCoreApp, self).__init__(import_name)
   localDefConfig = { 'DEBUG': True, 'FSDB_PATH': "", 'SECRET_KEY': 'asd', 'ES_INDEXNAME': 'libreant' }
   self.config.update(localDefConfig)
   self.config.update(defConf)
...
```
and  ```LibreantViewApp``` will start like this:

```python
def __init__(self, import_name, defConf={}):
   localDefConf = {'PRESET_PATHS': [], 'BOOTSTRAP_SERVE_LOCAL': True, 'AGHERANT_DESCRIPTION':[] }
   super(LibreantViewApp, self).__init__(import_name, localDefConf.update(defConf))
   ...
```